packages/nova-forms/lib/Form.jsx

```diff
@@ -333,6 +336,22 @@ class Form extends Component {
       errors: _.uniq([...prevState.errors, error])
     }));
   }
+  
+  // add to cleared fields
+  clearField(fieldName){
+    this.setState({
+      deletedFields: [...this.deletedFields, fieldName]
+    });
+    
+    return () => this.removeFromDeletedfields(fieldName);
```

Comment:
this pattern from event subscription, for example

var unsubscriber = eventEmitter.subcribe(...); I suppose it could be helpful.